common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -655,6 +662,10 @@ div.problem {
       font-style: italic;
       -webkit-font-smoothing: antialiased;
     }
+
+    @media print {
```

Comment:
This change hides all problem action buttons in the printing. Buttons include "Save", "Check", "Show/Hide Answer", etc inside problems.